<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QPushButton" name="btnSelectWindow">
        <property name="text">
         <string>获取窗口句柄(鼠标点击)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblHandleInfo">
        <property name="text">
         <string>未选择窗口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRunHshj">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>魂兽幻境副本</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="resultLayout">
      <item>
       <widget class="QLabel" name="lblMatchResult">
        <property name="text">
         <string>模板匹配坐标: -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblOcrResult">
        <property name="text">
         <string>OCR坐标: -</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblScreenshot">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>220</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="text">
       <string></string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
